src/libs/actions/Report.js

```diff
@@ -425,6 +417,65 @@ function removeOptimisticActions(reportID) {
     });
 }
 
+/**
+ * If an iouReport is open (has an IOU, but is not yet paid) then we maintain a link between it and the associated
+ * chatReport in Onyx, simplifying IOU data retrieval and reducing necessary API calls when displaying IOU components:
+ * - chatReport: {id: 123, iouReportID: 987, ...}
+ * - iouReport: {id: 987, chatReportID: 123, ...}
+ *
+ * This function allows us to fetch an iouReport without updating the linked chatReport, preventing the chatReport's
```

Comment:
NAB, IMO a doc should describe what a function does not what it does not do

Reply:
The problem here is that if I don't include information about the link, it is tricky to explain why this function needs to exist. I'm not sure how to remove the link info without making this harder to understand.

Reply:
Removed the large comment.

Reply:
Removed this.